libs/api/Verifier.cpp

```diff
@@ -139,15 +140,37 @@ verify_byte_code(
 {
     const ebpf_platform_t* platform = &g_ebpf_platform_windows;
     std::vector<ebpf_inst> instructions{
-        (ebpf_inst*)byte_code, (ebpf_inst*)byte_code + byte_code_size / sizeof(ebpf_inst)};
-    program_info info{platform};
+        (ebpf_inst*)byte_code, (ebpf_inst*)byte_code + byte_code_size / sizeof(ebpf_inst) };
+    program_info info{ platform };
     info.type = platform->get_program_type(section_name, path);
 
     raw_program raw_prog{path, section_name, instructions, info};
 
     return analyze(raw_prog, error_message);
 }
 
+int
+verify_byte_code2(
```

Comment:
Delete this and update the verify_byte_code() prototype to take program_type as an optional param.

Reply:
Changed this to verify_byte_code() and the other function to verify_byte_code_with_section()

Reply:
Now there is only one function -- verify_byte_code() which is used everywhere.